### DISCUSSION TITLE
Feature: Adjustable weighting of datasets

### DISCUSSION THREAD

**Discussion by viochemist** [gbouvignies/ChemEx#248] [Ideas]
This is more a point of conversation. Would it be worth having the option to up- or down-weight datasets? In general, Chi2 is going to be dependent on the number of points in a profile, so a CPMG profile with 20 nu_cpmg values could be ~20/residue, a CEST profile with 100 offsets would be ~100/residue, and hsqc/hmqc shifts would be about ~1/residue. Similarly, fitting 2-3 CEST B1 fields will generally have fewer points for higher B1 fields, leading to lower Chi2. In the end, this leads to variation in the numerical significance of each dataset. I honestly don't know if this matters or not, but I do see various fitted parameters dependent on the choice of dataset. I'd be curious about any thoughts on this.

**Comment by gbouvignies** [comment]
I guess this could be a parameter in the `[data]` section of experiment.toml files: 1.0 by default. This would affect the chi2 though making it more or less meaningless.
